src/Razor/src/Directory.Build.props

```diff
@@ -6,6 +6,7 @@
     <IsPackable>true</IsPackable>
     <IsShipping>true</IsShipping>
     <IncludeSymbols>true</IncludeSymbols>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
Needed to add this, because as it turns out there were things that the IDE would previously error on, but `build.cmd` wouldn't complain about at all. This has always annoyed me, but I never got around to investigating. With this set, the errors in command line now match the IDE, as do their severity (except in the compiler where this is turned off, and in tests where I didn't turn it on because there were too many errors 😛)

Reply:
.. and of course this has broken other things 🤦‍♂️